monitors/test_settings.py

```diff
@@ -22,12 +22,6 @@
 LAST_RUN_LOC = os.path.join('logs', 'lastrun.txt')
 EORM_LOG_FILE = os.path.join(get_project_root(), 'logs', 'end_of_run_monitor.log')
 EORM_LAST_RUN_FILE = os.path.join(get_project_root(), 'logs', 'eorm_last_runs.csv')
-INSTRUMENTS = [{'name': 'WISH', 'use_nexus': True},
```

Comment:
@dtasev well spotted. In fact looking more closely could all in this test_settings.py be deleted accept for setting CYCLE_FOLDER and LAST_RUNS_CSV?